pyproject.toml

```diff
@@ -1,3 +1,6 @@
+[tool.black]
+exclude = "/(\\.direnv|\\.eggs|\\.git|\\.hg|\\.mypy_cache|\\.nox|\\.tox|\\.venv|venv|\\.svn|\\.ipynb_checkpoints|_build|buck-out|dist|__pypackages__)/"
```

Comment:
The files in `tools/devctr/` are only used during building of our docker containers and should probably stay in there. Could you instead create a new `pyproject.toml` in the root?

Also, what are the disadvantages to just doing `black . --check --diff --exclude ''` over in https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/style/test_python.py? Testing locally that seems to give the same behavior.

Reply:
@roypat 
Thank you for review.

>The files in tools/devctr/ are only used during building of our docker containers

Devcontainer is convenient for quick setup.
But, I prefer use host environment directly because of some IDE support.
Since there are cases like this, I think that `pyproject.toml` should be placed in the project root without dividing.

>what are the disadvantages to just doing black . --check --diff --exclude ''

If you modify the code in `test_python.py`, this will only affect when you run the test code.
But, according #3166 , She try to use `black` on shell.
By adding settings to `pyproject.toml`, it is possible to apply common settings to both when executing test code and executing black directly on the shell.

Reply:
> If you modify the code in test_python.py, this will only affect when you run the test code.

Ah, I see, yes that makes sense!

Reply:
We already have some python config files in the `tests` directory, so it would be more consistent to place it there or (probably simpler) just create a new `pyproject.toml` there and leave the poetry setting close to the `Dockerfile`

Reply:
Created new `pyproject.toml` under `tests` directory.